common/beerocks/btl/CMakeLists.txt

```diff
@@ -21,6 +21,10 @@ set_target_properties(${PROJECT_NAME} PROPERTIES VERSION ${prplmesh_VERSION} SOV
 target_link_libraries(${PROJECT_NAME} PUBLIC bcl tlvf elpp btlvf common)
 if (NOT MSGLIB STREQUAL "None")
```

Comment:
Good - so this should be a fixup commit.

Reply:
Sure.